website/docs/deployment.mdx

```diff
@@ -543,13 +546,16 @@ jobs:
 
 <summary>Site not deployed properly?</summary>
 
-After pushing to main, if you don't see your site published at the desired location (for example, it says "There isn't a GitHub Pages site here", or it's showing your repo's README.md file), check the following:
+After pushing to main, if you don't see your site published at the desired location (for example, it says "There isn't a GitHub Pages site here," or it's showing your repo's README.md file), try the following:
+
+- Wait about three minutes and refresh. It may take a few minutes for GitHub pages to pick up the new files.
+- Check your repo's landing page for a little green tick next to the last commit's title, indicating the CI has passed. If you see a cross, it means the build or deployment failed, and you should check the log for more debugging information.
+- Click on the tick and make sure you see a "Deploy to GitHub Pages" workflow. Names like "pages build and deployment / deploy" are GitHub's default workflows, indicating your custom deployment workflow failed to be triggered at all. Make sure the YAML files are placed under the `.github/workflows` folder, and that the trigger condition is set correctly (e.g., if your default branch is "master" instead of "main," you need to change the `on.push` property).
+- Make sure the "Source" (which is the source for the _deployment_ files, not "source" as in our terminology) is set to "gh-pages" + "/ (root)" Under your repo's Settings > Pages (We are using `gh-pages` as the deployment branch).
```

Comment:
```suggestion
- Under your repo's Settings > Pages, make sure the "Source" (which is the source for the _deployment_ files, not "source" as in our terminology) is set to "gh-pages" + "/ (root)", since we are using `gh-pages` as the deployment branch.
```

I prefer the words to follow the steps users take (i.e. they first navigate to the tab and _then_ check the settings)